topi/python/topi/cuda/nms.py

```diff
@@ -28,27 +28,46 @@
 from .. import tag
 
 
-def get_valid_counts_pre(data, flag, idx, score_threshold, id_index, score_index):
-    """Low level IR to Prepare get valid count of bounding boxes
-    given a score threshold. Also moves valid boxes to the
+def cuda_atomicAdd_rule(op):
+    if op.dtype == "float32":
+        return tvm.call_pure_extern("float32", "atomicAdd", op.args[0], op.args[1])
+    elif op.dtype == "float64":
+        return tvm.call_pure_extern("float64", "atomicAdd", op.args[0], op.args[1])
+    elif op.dtype == "int32":
+        return tvm.call_pure_extern("int32", "atomicAdd", op.args[0], op.args[1])
+    else:
+        raise RuntimeError("only support int32, float32 and float64")
+
+
+tvm.target.intrin.register_intrin_rule(
+    "cuda", "atomicAdd", cuda_atomicAdd_rule, override=True)
+
+
+def atomicAdd(x, y):
+    return tvm.call_pure_intrin(y.dtype, "atomicAdd", x, y)
+
+
+def get_valid_counts_ir(data, valid_count, Flag, score_threshold, id_index, score_index):
```

Comment:
Change variable/function names to fit conventions
`Flag` -> `flag`
`atomicAdd` -> `atomic_add`
etc